src/rad/resources/schemas/reference_files/roman_skycells-1.0.0.yaml

```
@@ -0,0 +1,201 @@
+%YAML 1.1
+---
+$schema: asdf://stsci.edu/datamodels/roman/schemas/rad_schema-1.0.0
+id: asdf://stsci.edu/datamodels/roman/schemas/reference_files/roman_skycells-1.0.0
+
+title: Roman Skycells File Schema
```

```suggestion
title: Skycells File Schema
```
While looking at the roman datamodels PR I noticed the roman prefixes there. Do we need these?
Same goes for the name of this file. Is `skycells-1.0.0.yaml` sufficient?